### PR TITLE
task(issue-111): added button link in job page for viewing backlog

### PR DESCRIPTION
**Pull request by abirshah**

### CODE REVIEW

**Review by CeliaCaii — CHANGES_REQUESTED** [review]
The backlog button placement is very random in the Job page. View picture for a suggested placement of where the backlog button could be placed:
image

**Review by CeliaCaii — COMMENTED** [review]
The backlog button redirects me to this new page, but I believe we said it should be a pop up modal (similar to the add a bicycle button in Inventory page). However, I think that opening up a new page is also fine, but everyone should agree with it.

image

File: ERP/resources/views/job-backlog-page.blade.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@extends('layouts.master')
+@section('inside-body-tag')
+<div class="container-fluid">
```
Comment (pirasunt) [inline ERP/resources/views/job-backlog-page.blade.php:3]: @abirshah Do you think you can add this part to a MODAL in the jobs page?

**Review by pirasunt — CHANGES_REQUESTED** [review]

File: ERP/resources/views/job-backlog-page.blade.php (hunk 1 above)
Comment (pirasunt) [reply ERP/resources/views/job-backlog-page.blade.php]: @abirshah Looks great now man! Thanks
- However, i think you forgot to remove the old backlog button
image

**Comment by pirasunt** [comment]
@abirshah I think you forgot to remove the old backlog button.
image

**Review by pirasunt — APPROVED** [review]
Awesome job!!

**Review by CeliaCaii — APPROVED** [review]

**Pull request merged**